src/app/components/workspace/registry/component-registry.ts

```diff
@@ -35,21 +35,21 @@ export class ComponentRegistry {
     /**
      * Extracts the Components from the GL's component hierarchy.
      *
-     * @param content GL's "content" attribute which contains a tree of rows/columns/components
+     * @param content GL's "tabData" attribute which contains a tree of rows/columns/components
```

Comment:
Again, overzealous search and replace? Also, do we still need the whole workspace component and accompanying services?